AnatomyOfMatplotlib-Part4-Limits_Legends_and_Layouts.ipynb

```diff
@@ -340,7 +348,51 @@
      ],
      "language": "python",
      "metadata": {},
-     "outputs": []
+     "outputs": [],
+     "prompt_number": null
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Another common "
```

Comment:
Ah, sorry that slipped in there. I accidentally applied a stash and forgot I'd applied it...